apps/web/src/app/page.tsx

```
@@ -24,183 +26,196 @@ export default async function HomePage() {
   const cookieStore = await cookies();
   const refreshToken = cookieStore.get('refreshToken');
   const isLoggedIn = !!refreshToken;
+  const featuresContent = [
+    <div>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)